code/game/machinery/computer/arcade.dm

```diff
@@ -571,23 +571,24 @@
 		event = null
 
 	else if(href_list["killcrew"]) //shoot a crewmember
-		var/sheriff = remove_crewmember() //I shot the sheriff
-		playsound(loc,'sound/weapons/gunshots/gunshot.ogg', 100, 1)
+		if(settlers.len > 0 || alive > 0)
```

Comment:
An early return is preferred in our codebase. In this case it'll be something like this:
```dm
if(amount <= 0 || alive <= 0)
  return
do_happy_flow
```
Also be sure to use `length(X)` instead of `X.len`. `length(X)` is null pointer exception proof and faster compared to `X.len`